features.json

```diff
@@ -53,6 +53,7 @@
     "MP": true,
     "TIKTOK_ADS_OFFLINE_EVENTS": true,
     "CRITEO_AUDIENCE": true,
-    "CUSTOMERIO": true
+    "CUSTOMERIO": true,
+    "BINGADS_AUDIENCE": true
```

Comment:
why router transform?

Reply:
For OAuth we need it right?

Reply:
we are not delivering using transformer, we are delivering using server, @sanpj2292 can you comment on this?

Reply:
[Q]: Why we prefer router transform for OAuth destinations ? 
[A]: There can be API calls during transformation for which we need token information & rt to delivery is not a time-taking process usually, so the chances of token expiry before delivery is less.

Questions for @koladilip
1:  Do we have a way to parse response and make sure we trigger refresh flow only for specific error(s) ?
2: Transformation is not required for this use-case. Am I right ?! Or Am I missing something here ?

**Point to note**:
The important thing for `OAuth destinations` is we need to do parse the response & identify if the error indicates we need to refresh the token, if we can do that we're good with not doing router transformation.

Reply:
We are going to deliver and handle oauth stuff completely in server so not sure this is needed

Reply:
No transformation required ?